locales/es/pokemon.json

```diff
@@ -54,32 +54,35 @@
       }
     },
     "karp": {
-      "orange": "",
-      "white": "",
-      "gold": "",
-      "brown": "",
-      "pink": "",
-      "blue": "",
-      "grey": "",
-      "purple": "",
-      "apricot": "",
+      "orange": "Rojo",
+      "orange_before": "Roji",
+      "orange_after": "rojo",
+      "white_after": "blanco",
+      "white_before": "Blanqui",
```

Comment:
I think the best way to handle these is to just write them in the actual translation as you did with Roja, Negra etc.
Too many variations to account for and we can't have extra keys in just some files with the testing setup I have done.